### PR TITLE
.NET API readme

### PR DESCRIPTION
**Pull request by yarmolovichalex**

### CODE REVIEW

File: services/api-dotnet/README.md [hunk 1]
```diff
@@ -1,50 +1,53 @@
...
+- configured console logging ([Serilog](https://serilog.net/))
+- request data validation ([FluentValidation](https://fluentvalidation.net/))
+- object mapping ([Automapper](https://automapper.org/))
+- email sender implementation for local development (can be used with [smtp4dev](https://github.com/rnwood/smtp4dev), for example)
+- Redis cache (see [CacheController](src/app/Api.NoSql/Controllers/CacheController.cs) for example of usage)
+- health checks
+- [Hangfire](https://www.hangfire.io/) scheduler. Can be configure to use either MongoDB or PostgreSQL storage. Dashboard is available on http://localhost:3001/hangfire
+- Feature flags support ([FeatureManagement](https://github.com/microsoft/FeatureManagement-Dotnet)). See [FeatureController](src/app/Api.NoSql/Controllers/FeatureController.cs) for example of usage)
+- Docker support
+- production ready account API resource (singup, signin, forgot password, reset password functionality)
```
Comment (SergeyLukashov) [inline services/api-dotnet/README.md:32]: I'd move this point to the very top of the list as it's core of the project. Also believe it's beneficial to mention that API could be backed by Postgres and Mongo here too.

Comment (SergeyLukashov) [reply services/api-dotnet/README.md:32]: Another point that could be mentioned in this list is DAL. We provide not only naked API but also a bunch of services for convenient data access. Such as filtration, pagination, sorting, etc.

File: services/api-dotnet/README.md [hunk 2]
```diff
@@ -1,50 +1,53 @@
...
-2. [React.JS SPA](https://github.com/paralect/koa-react-starter) (and simple Node.JS server to server index.html file and static assets)
-3. [Deployment automation](https://github.com/paralect/ship/tree/master/deploy/app) with Docker & Ansible. 
-4. [CI automation](https://github.com/paralect/ship/tree/master/deploy/drone-ci) using Drone CI (we might use other CI if this won't work)
-5. [.NET project srtart via single command](https://github.com/paralect/docker-compose-starter). We plan to use self-hosted API service to being able to run it within Docker. 
+# .NET API starter
+
+.NET is a huge part of the Paralect history. In fact, for the first 5 years of the company .NET was a primary technology for most of the projects. We love technology and beleive it suits very well for what we do: product engineering.
+
+This is a fully featured .NET restful API starter application, which is configured to work with either MongoDB or PostgreSQL. The goal of this project is to solve all routine tasks and keep your focus on the product and business logic of the application, not on the common things, such as logging, configuration, dev/production environments.
```
Comment (SergeyLukashov) [inline services/api-dotnet/README.md:5]: Do you think it's worth adding version of the .NET? 

.NET -> .NET 6

File: services/api-dotnet/README.md (hunk 1 above)
Comment (SergeyLukashov) [reply services/api-dotnet/README.md:32]: And last - we can also put here build-in Unit Testing feature.

File: services/api-dotnet/README.md [hunk 3]
```diff
@@ -1,50 +1,53 @@
...
+|[Common](src/app/Common)|DAL (for MongoDB and PostgresSQL) and some configuration, which can be shared across all applications|
+|[Services](src/app/Common.Services)|Domain/infrastructure services, shared across all applications|
+|[Scheduler](src/app/Scheduler)|Hangfire scheduler|
+|[Websocket server](src/app/SignalR)|SignalR websocket server (works with MongoDB only)|
+|[Unit tests for NoSQL API](src/app/Tests.NoSql)|Unit tests for NoSQL API|
+|[Unit tests for SQL API](src/app/Tests.Sql)|Unit tests for SQL API|
+
+## Features
+
+- configured console logging ([Serilog](https://serilog.net/))
```
Comment (SergeyLukashov) [inline services/api-dotnet/README.md:23]: I'd capitalize all the points sentences.

File: services/api-dotnet/README.md [hunk 4]
```diff
@@ -1,50 +1,53 @@
...
-	- For mongo service replace "- /var/run/docker.sock:/var/run/docker.sock" with "- mongodata:/data/db" (The path /var/run/docker.sock does not exist on Windows)
-	- Remove api service entirely (.Net API server starts separately)
-	- Add mongodata volume declaration like "db-data" volume here:
-	https://docs.docker.com/compose/compose-file/#endpoint_mode
+Typically a new project will need either SQL or NoSQL DB, so it'll be necessary to make a few changes.
 
-2. web/src/server/config/environment/development.js:
-	- Change jwt secret setting to "jwt_secret128bits" (in .Net it needs to be more than 127 bist long)
+In case of MongoDB:
+- remove Api.Sql project
```
Comment (SergeyLukashov) [inline services/api-dotnet/README.md:42]: Numbered list fits here better to my mind. As this is kind of instruction, not raw list.

Comment (SergeyLukashov) [reply services/api-dotnet/README.md:42]: Also thinking if it makes sense to integrate the last point about `docker-compose.yml` and `appsettings.*.json` files into these instructions too 🤔

**Review by SergeyLukashov — COMMENTED** [review]
Great job! I didn't realize our readme got so outdated 🤯

Please take a look on the suggestions, address and later feel free to merge 🙏🏻

File: services/api-dotnet/README.md (hunk 4 above)
Comment (yarmolovichalex) [reply services/api-dotnet/README.md:42]: I'll remove most of these steps by configuring create-ship-app to build a new solution with relevant folders / docker-compose.yml / configs / etc.

**Pull request merged**